dashboard/app/views/home/index.html.haml

```diff
@@ -25,6 +25,10 @@
   - if current_user
     #user_hero
       = render template: 'api/user_hero'
+    - [current_user.working_on_user_scripts, current_user.completed_user_scripts].compact.reduce([], :|).each_slice(2) do |course_pair|
```

Comment:
No, I need it, so it "flattens" it too (otherwise I get undefined method 'script') Or I could use flatten? Not sure which is better.